addons/web/static/src/views/list/list_renderer.js

```diff
@@ -672,30 +701,32 @@ export class ListRenderer extends Component {
         }
         const classNames = [...this.cellClassByColumn[column.id]];
         if (column.type === "field") {
-            if (record.isRequired(column.name)) {
-                classNames.push("o_required_modifier");
-            }
-            if (record.isInvalid(column.name)) {
-                classNames.push("o_invalid_cell");
-            }
-            if (record.isReadonly(column.name)) {
-                classNames.push("o_readonly_modifier");
-            }
-            if (this.canUseFormatter(column, record)) {
-                // generate field decorations classNames (only if field-specific decorations
-                // have been defined in an attribute, e.g. decoration-danger="other_field = 5")
-                // only handle the text-decoration.
-                const { decorations } = record.activeFields[column.name];
-                for (const decoName in decorations) {
-                    if (evaluateExpr(decorations[decoName], record.evalContext)) {
-                        classNames.push(getClassNameFromDecoration(decoName));
+            if (!column.propertyType) {
```

Comment:
can we merge those 2 conditions to reduce the diff ?

Reply:
I'll just make it an ``if-elif`` and set all the classes that would be set for a regular read-only field